### ISSUE TITLE
English Translation

### ISSUE THREAD

**Issue by Sinevia** [gogf/gf-doc#53]
Would be good this framework to be translated to English

**Comment by gqcn** [comment]
@Sinevia Yep, the go doc is in English: https://godoc.org/github.com/gogf/gf . Any contribution of English documentation for usage of this framework will be appreciated.

**Comment by Sinevia** [comment]
Thanks @gqcn 

I was talking about the website: https://goframe.org/index . Cannot see any translations on  it.

The framework looks promising by looking at the images and code examples mainly.

**Comment by gqcn** [comment]
@Sinevia Thanks for your attention to GoFrame, but there's no recent planning for English translation support yet. Maybe next year. But if there's any contribution for English translation, it'd be quite appreciated.
